src/gui/properties/speedplotview.h

```diff
@@ -0,0 +1,89 @@
+#ifndef SPEEDPLOTVIEW_H
+#define SPEEDPLOTVIEW_H
+
+#include <QGraphicsView>
+#include <QPen>
+#include <QQueue>
+#include <QMap>
+
```

Comment:
```
In file included from gui/properties/speedplotview.cpp:1:0:
gui/properties/speedplotview.h:6:7: error: template argument required for 'class QMap'
 class QMap;
       ^
gui/properties/speedplotview.h:41:5: error: 'QQueue' is not a template
     QQueue<double> m_xData;
     ^
gui/properties/speedplotview.h:41:20: error: field 'm_xData' has incomplete type 'QQueue'
     QQueue<double> m_xData;
                    ^
gui/properties/speedplotview.h:42:15: error: 'QQueue' is not a template
     QMap<int, QQueue<double> > m_yData;
....
```
Only for QPen without errors. I think that because QMap and QQueue are templates.